OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/OptimoveConfig.java

```diff
@@ -193,31 +247,47 @@ public DeferredDeepLinkHandlerInterface getDeferredDeepLinkHandler() {
     }
 
     public boolean isOptimoveConfigured(){
-        return this.optimoveToken != null && this.configFileName != null ;
+        // TODO: optimove part is not ready for this yet
+        // this.featureSet == FeatureSet.OPTIMOVE_ONLY || this.featureSet == FeatureSet.ALL
+        return this.optimoveToken != null && this.configFileName != null;
```

Comment:
Do we have any benefit from the feature set now? Seems like it is inferred from whether credentials exist or not. Is something else planned for this enum?

Reply:
there are 2 constructors:
1. old one: init(optimove, optimobile) --> featureSet inferred
2. new one: init(region, featureSet) --> users tell us which features do they want, so, we can tell which parts to partially initialize before they give credentials. (Partial init of optimove is not possible in the 1st pass, but optimobile is)